lib/release-please/node-workspace.js

```diff
@@ -81,9 +81,13 @@ module.exports = class extends NodeWorkspace {
       for (const [depName, resolved] of graphPackage.localDependencies) {
         const depVersion = updatedVersions.get(depName)
         const isNotDir = resolved.type !== 'directory'
-        // Changelog entries are only added for dependencies and not any other type
-        const isDep = Object.prototype.hasOwnProperty.call(pkg.dependencies, depName)
-        if (depVersion && isNotDir && isDep) {
+
+        // Due to some bugs in release-please we need to create
+        // deps commits for all dev and prod dependencies even though
+        // we normally would only do a `chore:` commit to bump a dev
+        // dep. The tradeoff is an extra path to our workspaces that
+        // depend on other workspaces as dev deps.
```

Comment:
> The tradeoff is an extra path to our workspaces that depend on other workspaces as dev deps.

I don't understand what this means.

Reply:
There's a typo in there, plus I think i got the devdep relationship backwards. It should read: `The tradeoff is an extra patch release to workspaces that are depended on only as dev deps.`

Reply:
This is why https://github.com/npm/cli/actions/runs/3185212191/jobs/5194519454 is failed. It is attempting to bump `libnpmpublish` since it's devdep `libnpmpack` is getting released, but `libnpmpublish` has no other commits so it blows up. This change forces `libnpmpublish` to get a `deps: libnpmpack` commit.